root/static/scripts/relationship-editor/generic.js

```diff
@@ -168,6 +161,12 @@ import {SERIES_ORDERING_TYPE_AUTOMATIC} from '../common/constants';
                 }
             });
 
+            _.each(['entity0_credit', 'entity1_credit'], function (prop) {
+                if (typeof changeData[prop] === 'string') {
```

Comment:
This makes the code consistent with its surroundings, by only outputting fields that were changed.